--- edittableform.cpp.str	2010-02-18 08:31:01.000000000 +0100
+++ edittableform.cpp	2010-02-18 08:31:13.000000000 +0100
@@ -86,7 +86,7 @@
        QApplication::setOverrideCursor( Qt::waitCursor ); // this might take time
        modified = true;
        QString newName = renTableForm->getTableName();
-       qDebug(newName);
+       qDebug("%s", newName.latin1() );
        QString sql;
        //do the sql rename here
        //if (!pdb->executeSQL(QString("BEGIN TRANSACTION;"))) goto rollback;
